.github/workflows/benchmarks_compute.yml

```diff
@@ -167,6 +167,14 @@ jobs:
       id: benchmarks
       run: numactl -N 0 ${{ github.workspace }}/ur-repo/scripts/benchmarks/main.py ~/bench_workdir ${{github.workspace}}/sycl_build ${{github.workspace}}/ur-repo ${{ matrix.adapter.str_name }} ${{ inputs.bench_script_params }}
 
+    - name: Upload results
```

Comment:
This doesn't seem like a good idea from a security point of view. How is this git authenticated?
This should work similarly to how we publish documentation, see this existing workflow: https://github.com/oneapi-src/unified-runtime/blob/main/.github/workflows/docs.yml

Reply:
In final solution there are no changes in .yml files